ui/app/src/components/PathNavigatorTree/PathNavigatorTreeItem.tsx

```diff
@@ -180,6 +182,9 @@ const useStyles = makeStyles((theme) =>
     },
     iconButton: {
       padding: '2px 3px'
+    },
+    active: {
+      backgroundColor: palette.blue.highlight
```

Comment:
use theme's active setting

Reply:
The one matching the blue color is theme.palette.action.selected, the `active` one is a dark gray